tl-tl/sc/test.md

```diff
@@ -59,33 +59,33 @@ namespace ConsoleApplication1
 
             using (ScriptBuilder sb = new ScriptBuilder())
             {
-                sb.EmitPush(2); // corresponds to the parameter c
-                sb.EmitPush(4); // corresponds to the parameter b
-                sb.EmitPush(3); // corresponds to the parameter a
+                sb.EmitPush(2); // tumutugma sa c na parametero
+                sb.EmitPush(4); // tumutugma sa b na parametero
+                sb.EmitPush(3); // tumutugma sa a na parametero
                 engine.LoadScript(sb.ToArray());
             }
 
-            engine.Execute(); // start execution
+            engine.Execute(); // simulang ang eksekyusyon
 
-            var result = engine.EvaluationStack.Peek().GetBigInteger(); // set the return value here
+            var result = engine.EvaluationStack.Peek().GetBigInteger(); // dito itakda ang isasauling halaga
             Console.WriteLine($"Execution result {result}");
             Console.ReadLine();
         }
     }
 }
 ```
 
-Compile output: Execution result 14, as expected
+Output sa pag-compile: Resulta ng eksekyusyon ay 14, tulad ng inaasahan
 
-> [!Note]
+> [!Tandaan]
 >
-> If you encounter the following error after execution：
+> Kung matatagpo ang sumusunod na error pagkatapos mag-eksekyut:
 >
-> The type “BigInteger” is defined in an unreferenced assembly. You must add a reference to the assembly “System.Numerics, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089”
+> Ang tipo na “BigInteger” ay i-denifine sa isang unreferenced na assembly. Kinakailangan mong magdagdag ng reference sa assembly na “System.Numerics, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089”
 >
-> You can add a reference to “System.Numerics”  to solve the problem。
+> Pwede kang magdagdag ng isang reference sa “System.Numerics” para maresolba ang problema。
 
-Note: If you use the above code to pass the parameters, pay attention to the top of the stack corresponding to the first parameter, for convenience you can also pass the parameters with following code.S
+Tandaan: Kung gagamit ka ng code sa itaas para pumasa ng mga parametero, bigyang pansin ang nasa taas ng stack na naaayon sa unang parametro, pwede ring ipasa ang mga parametro gamit ang sumusunod para maging madali.S
```

Comment:
- `parametero` to `parametro`
- `sumusunod para maging madali sa pamamagitan ng sumusunod na code`

Reply:
fix provided